### PR TITLE
feat(deletions): Backfill group_id column

### PR DESCRIPTION
**Pull request by lynnagara**
https://github.com/getsentry/sentry/pull/14972 and https://github.com/getsentry/sentry/pull/14988 should be merged first

### CODE REVIEW

File: src/sentry/south_migrations/0514_backfill_attachment_group_id.py [hunk 1]
```diff
@@ -0,0 +1,1534 @@
...
+
+    def _forwards(self, orm):
+        "Write your forwards methods here."
+        # Note: Don't use "from appname.models import ModelName".
+        # Use orm.ModelName to refer to models in this application,
+        # and orm['appname.ModelName'] for models in other applications.
+        EventAttachment = orm['sentry.EventAttachment']
+        Event = orm['sentry.Event']
+
+        queryset = EventAttachment.objects.all()
```
Comment (markstory) [inline src/sentry/south_migrations/0514_backfill_attachment_group_id.py]: Should you exclude EventAttachments that have a group_id? That would help make this migration idempotent.

**Review by markstory — APPROVED** [review]

File: src/sentry/south_migrations/0514_backfill_attachment_group_id.py (hunk 1 above)
Comment (lynnagara) [reply src/sentry/south_migrations/0514_backfill_attachment_group_id.py]: Good idea, thanks

File: src/sentry/south_migrations/0514_backfill_attachment_group_id.py [hunk 2]
```diff
@@ -0,0 +1,1534 @@
...
+    is_dangerous = True
+
+    def forwards(self, orm):
+        db.commit_transaction()
+        try:
+            self._forwards(orm)
+        except Exception:
+            db.start_transaction()
+            raise
+        db.start_transaction()
```
Comment (fpacifici) [inline src/sentry/south_migrations/0514_backfill_attachment_group_id.py]: finally ?

File: src/sentry/south_migrations/0514_backfill_attachment_group_id.py [hunk 3]
```diff
@@ -0,0 +1,1534 @@
...
+
+    def _forwards(self, orm):
+        "Write your forwards methods here."
```
Comment (fpacifici) [inline src/sentry/south_migrations/0514_backfill_attachment_group_id.py]: I think you actually accomplished this.

**Review by fpacifici — APPROVED** [review]

File: src/sentry/south_migrations/0514_backfill_attachment_group_id.py (hunk 2 above)
Comment (lynnagara) [reply src/sentry/south_migrations/0514_backfill_attachment_group_id.py]: Hmmm, I don't want to change this autogenerated stuff, this seems to be present in the other data migrations I've looked at.

Comment (fpacifici) [reply src/sentry/south_migrations/0514_backfill_attachment_group_id.py]: ok, if it was autogenerated ignore my comment

**Pull request closed without merging**